packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -279,73 +283,98 @@ export class VaPagination {
-    const nextAriaLabel = ariaLabelSuffix ? `Next page ${ariaLabelSuffix}` : 'Next page';
-    const lastPageAriaLabel = ariaLabelSuffix ? `Page ${pages} ${ariaLabelSuffix}, last page` : `Page ${pages}, last page`;
+    const previousAriaLabel = ariaLabelSuffix
+      ? `Previous page ${ariaLabelSuffix}`
+      : 'Previous page';
+    const nextAriaLabel = ariaLabelSuffix
+      ? `Next page ${ariaLabelSuffix}`
+      : 'Next page';
+    const lastPageAriaLabel = ariaLabelSuffix
+      ? `Page ${pages} ${ariaLabelSuffix}, last page`
+      : `Page ${pages}, last page`;
     if (uswds) {
       const pageNumbersToRender = this.pageNumbersUswds();
       const itemClasses = classnames({
         'usa-pagination__item': true,
         'usa-pagination__page-no': true,
-        'va-pagination__item': true
+        'va-pagination__item': true,
       });
       const ellipsisClasses = classnames({
         'usa-pagination__item': true,
         'usa-pagination__overflow': true,
-        'va-pagination__item': true
+        'va-pagination__item': true,
       });
       const arrowClasses = classnames({
         'usa-pagination__item': true,
         'usa-pagination__arrow': true,
-
       });
 
-      const previousButton = page > 1
-        ?
-        <Fragment>
-          <li class={arrowClasses} aria-label={previousAriaLabel}>
-            <a
-              onClick={() => this.handlePageSelect(page - 1, 'nav-paginate-number')}
-              onKeyDown={e => this.handleKeyDown(e, page - 1)}
-              class="usa-pagination__link usa-pagination__previous-page"
-              href="javascript:void(0)"
-            >
-              <div id="previous-arrow-icon"></div>
-              <span class="usa-pagination__link-text">{i18next.t('previous')}</span>
-            </a>
-          </li>
-          {!pageNumbersToRender.includes(1) &&
+      const previousButton =
+        page > 1 ? (
           <Fragment>
-            <li class={itemClasses}>
+            <li class={arrowClasses} aria-label={previousAriaLabel}>
               <a
-                onClick={() => this.handlePageSelect(1, 'nav-paginate-number')}
-                onKeyDown={e => this.handleKeyDown(e, 1)}
+                onClick={() =>
+                  this.handlePageSelect(page - 1, 'nav-paginate-number')
+                }
+                onKeyDown={e => this.handleKeyDown(e, page - 1)}
+                class="usa-pagination__link usa-pagination__previous-page"
                 href="javascript:void(0)"
-                class="usa-pagination__button"
-                aria-label="page 1, first page"
-                >1</a>
-            </li>
-            <li class={ellipsisClasses} aria-label="ellipsis indicating non-visible pages">
-              <span>…</span>
+              >
+                <div id="previous-arrow-icon"></div>
+                <span class="usa-pagination__link-text">
+                  {i18next.t('previous')}
+                </span>
+              </a>
```

Comment:
This is a little difficult to review because there's a lot of formatting changes. Did something need to change in the structure of `va-pagination` that makes it diverge from the [USWDS example](https://designsystem.digital.gov/components/pagination/)? Or is all of this formatting updates?

Reply:
@jamigibbs The only actual change should be line 33. adding: `assetsDirs: ['../../assets']` to the component decorator. That's to help stencil link the component and the assets it needs:
https://stenciljs.com/docs/assets#assetsdirs